tidb-cloud/migration-job.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Migrate Online or On-Premises Data Using Migration Jobs
```

Comment:
does the online mean "on cloud"? if it means no downtime, it's a bit weird that what the meaning of "or". We support "no downtime + cloud", "no downtime + op", "downtime + op"?

Reply:
Good point. Can we change it to: Migrate On-Cloud and On-Premises Data Using Migration Jobs

Reply:
Updated